transformation/transformationPlugins/jsonMessageTransformers/jsonTypeMappingsSanitizationTransformer/src/main/resources/js/typeMappingsSanitizer.js

```diff
@@ -5,7 +5,7 @@ function isEnabled(features, path) {
     const keys = path.split('.');
 
     for (const key of keys) {
-        if (value && typeof value === 'object' && key in value) {
+        if (value && typeof value === 'object' && value.has("key")) {
```

Comment:
Will the former still work or no?  Does this mean that we're losing the map to javascript first class object wrapping?